app-android/src/main/java/com/ustadmobile/port/android/view/ClazzEditFragment.kt

```diff
@@ -284,4 +317,393 @@ class ClazzEditFragment() : UstadEditFragment<ClazzWithHolidayCalendarAndSchoolA
     }
 
 
+}
+
+private val CONTENT_ENTRY_TYPE_ICON_MAP = mapOf(
```

Comment:
This also needs to include the icon maps for CourseBlock as per CourseBlock.kt:

        const val BLOCK_MODULE_TYPE = 100

        const val BLOCK_TEXT_TYPE = 102

        const val BLOCK_ASSIGNMENT_TYPE = 103

        const val BLOCK_CONTENT_TYPE = 104

        const val BLOCK_DISCUSSION_TYPE = 105